<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1632</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>A4文字轉G-Code產生器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="bt_gen">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微軟正黑體</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>產生G-Code</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>繪製速度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_drawspeed">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>4000</string>
          </property>
          <property name="cursorPosition">
           <number>4</number>
          </property>
          <property name="placeholderText">
           <string>繪製速度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>空跑速度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_travelspeed">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>5000</string>
          </property>
          <property name="placeholderText">
           <string>空跑速度</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>文字大小</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sl_text_size">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>四邊留白</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sl_padding">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string> X軸間距</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sl_xspace">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>  10</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string> Y軸間距</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sl_yspace">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>  10</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="input_area">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微軟正黑體</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>請輸入內容</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGraphicsView" name="preview_window">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微軟正黑體</family>
          <pointsize>16</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="bt_pre_page">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>上一頁</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>共0/0頁</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_next_page">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>下一頁</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="num_lens_gcode">
        <property name="font">
         <font>
          <family>微軟正黑體</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>行數：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="gcode_window">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>16</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1632</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sl_text_size</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_8</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>671</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sl_padding</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_9</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>650</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sl_xspace</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_10</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>652</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sl_yspace</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_11</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
